openverse_catalog/dags/common/popularity/sql.py

```diff
@@ -267,12 +289,19 @@ def create_media_popularity_constants_view(
 
 def update_media_popularity_constants(
     postgres_conn_id,
+    task,
     media_type=IMAGE,
     popularity_constants_view=IMAGE_POPULARITY_CONSTANTS_VIEW,
 ):
     if media_type == AUDIO:
         popularity_constants_view = AUDIO_POPULARITY_CONSTANTS_VIEW
-    postgres = PostgresHook(postgres_conn_id=postgres_conn_id)
+    # TO DO see note above about when the materialized view gets populated. Not sure how
+    # often it is refreshed verses instantiated from scratch. And whether / how we might
+    # improve performance by integrating index updates/builds with that schedule.
```

Comment:
It's only re-instantiated when we make a change to the schema that would require recreating it I believe. The refresh is run _much_ more frequently, I believe monthly as part of the data refresh now!